packages/components/src/components/accordion/_accordion.scss

```diff
@@ -15,6 +15,19 @@
 /// @access private
 /// @group accordion
 @mixin accordion {
+  /// Accordion type tokens that can be overwritten
+  /// @type Map
+  /// @access public
+  /// @group accordion
+  $accordion-type-tokens: $tokens !default !global;
```

Comment:
Is this so that we can do the lookup by string? Is there anyway we could pass the variable directly instead or is something preventing that?

Reply:
This just so you could isolate accordion type token overrides. E.g. in your build:

```scss
$accordion-type-tokens: map-merge(
  $tokens,
  (
    'body-long-01': $body-long-02,
  )
);
$accordion-heading-height: $carbon--spacing-09;
@import '~carbon-components/src/components/accordion/accordion';

// button's `$tokens` unaffected by accordion overrides
@import '~carbon-components/src/components/button/button';
```

---

The other approach that would eliminate the need of a new component type token would be:

* Every component has dedicated `_mixins.scss`
* So you could import those mixins without outputting CSS
* Then you could:

```scss
@import '~carbon-components/src/components/accordion/mixins';

// override accordion's use of `body-long-01` so font-sizes and related properties are 16px instead of 14px
$my-tokens: map-merge(
  $tokens,
  (
    'body-long-01': $body-long-02,
  )
);

@include accordion($my-tokens);
```

* The mixin's token parameter would default to `$tokens` so the parameter would be optional
* Maybe the mixin parameter could be of type `argslist` so you could pass any override? Don't know.

Reply:
Recent commit removes the need for exposing a new public `$accordion-type-tokens` variable.